### PR TITLE
Replaced Time.now with <current date/time>

### LINKED ISSUE #30: Replace "Time.now" with better default value
`Time.now` is Ruby specific. Maybe we should do something like `2008-04-10T08:00:00Z (with current date/time)` or `<current time>`

### List your notifications

Param: since

https://developer.github.com/v3/activity/notifications/#list-your-notifications
        
### List your notifications in a repository

Param: since

https://developer.github.com/v3/activity/notifications/#list-your-notifications-in-a-repository
        
### Mark as read

Param: last_read_at

https://developer.github.com/v3/activity/notifications/#mark-as-read
        
### Mark notifications as read in a repository

Param: last_read_at

https://developer.github.com/v3/activity/notifications/#mark-notifications-as-read-in-a-repository

### PR DESCRIPTION
**Pull request by knrt10**
fixes #30

### CODE REVIEW

File: lib/endpoint/find-parameters.js [hunk 1]
```diff
@@ -216,3 +216,12 @@ function addPaginationParameters (state) {
...
 }
+
+function makeRelativeDefaultAbsolute (value) {
+  for (let i = 0; i < value.length; i++) {
+    if (value[i].default === 'Time.now') {
+      value[i].default = value[i].default.replace('Time.now', '<current date/time>')
+    }
+  }
+  return value
+}
```
Comment (gr2m) [inline lib/endpoint/find-parameters.js]: `makeRelativeDefaultAbsolute` is probably not a good name? Maybe do something like `replaceTimeNowDefault`? Also like in the other PR, passing in `state.results[0].params` is not enough, please pass in state like `findInBlocks` and the other methods above. Also please leave the empty line above the `// check for dublicate names` comment

**Comment by knrt10** [comment]
@gr2m I have to pass all methods to replaceTimeNowDefault ? Cause finally everything is pushed to result.parmas would it make a difference? Just asking cause not sure if any other case arises.

**Comment by knrt10** [comment]
@gr2m I updated the PR please check and review changes.

File: lib/endpoint/find-parameters.js [hunk 2]
```diff
@@ -216,3 +217,13 @@ function addPaginationParameters (state) {
     result.params.push(...PAGINATION_VARIABLES)
   })
 }
+
+function replaceTimeNowDefault (state) {
+  const results = state.results[0].params
```
Comment (gr2m) [inline lib/endpoint/find-parameters.js]: like the other PR, you have to iterate trough the `state.results` array, too

**Comment by knrt10** [comment]
@gr2m fixed the branch.

File: lib/endpoint/find-parameters.js [hunk 3]
```diff
@@ -227,3 +228,15 @@ function makeRelativeLinksAbsolute (state) {
...
+function replaceTimeNowDefault (state) {
+  state.results.forEach(output => {
+    for (let i = 0; i < output.params.length; i++) {
+      if (output.params[i].default === 'Time.now') {
+        output.params[i].default = output.params[i].default.replace('Time.now', '<current date/time>')
+      }
+    }
+    return output
+  })
+  return state
```
Comment (gr2m) [inline lib/endpoint/find-parameters.js]: you can remove both `return` statements, they don’t do anything. You use return for `.map()`, `.forEach()` doesn’t do anything with the returned value

File: lib/endpoint/find-parameters.js [hunk 4]
```diff
@@ -227,3 +228,15 @@ function makeRelativeLinksAbsolute (state) {
   })
   return state
 }
+
+function replaceTimeNowDefault (state) {
+  state.results.forEach(output => {
+    for (let i = 0; i < output.params.length; i++) {
+      if (output.params[i].default === 'Time.now') {
+        output.params[i].default = output.params[i].default.replace('Time.now', '<current date/time>')
```
Comment (gr2m) [inline lib/endpoint/find-parameters.js]: I think you can simplify this line. No need to use `.replace()`, you can just set the new `.default` to '<current date/time>'

File: lib/endpoint/find-parameters.js [hunk 5]
```diff
@@ -227,3 +228,15 @@ function makeRelativeLinksAbsolute (state) {
   })
   return state
 }
+
+function replaceTimeNowDefault (state) {
+  state.results.forEach(output => {
+    for (let i = 0; i < output.params.length; i++) {
```
Comment (gr2m) [inline lib/endpoint/find-parameters.js]: I don’t think you need to iterate using a `for () {}` loop here, did you try to use `output.params.forEach()`?

**Pull request merged**